app/components/obs/inputs/ObsBoolInput.vue.ts

```diff
@@ -1,13 +1,27 @@
 import { Component, Prop } from 'vue-property-decorator';
 import { ObsInput, IObsInput, TObsType } from './ObsInput';
+import VFormGroup from '../../shared/inputs/VFormGroup.vue';
```

Comment:
These imports should be absolute rather than relative:
`import VFormGroup from 'components/shared/inputs/VFormGroup.vue';`